src/components/seeds/database/TableCellRenderer.tsx

```diff
@@ -84,5 +84,10 @@ export default function SearchCellRenderer(props: RendererProps<SearchResponseEl
     return <CellRenderer index={index} column={column} value={`${value}%`} row={row} />;
   }
 
+  if (column.key === 'geolocations.coordinates') {
+    const coordinatesValue = ((row.geolocations || []) as any[]).map((gl) => gl.coordinates).join('; ');
+    return <CellRenderer index={index} column={column} value={coordinatesValue} row={row} />;
```

Comment:
This is great. Sorry I did not mention this earlier, but if there is a chance that this text might be too long for the table cell, you can use the `TextTruncated` component.

[Here's](https://github.com/terraware/terraware-web/blob/dd8702d485590e41233c7cbef0d5a9bf945ba7ea/src/scenes/Species/TableCellRenderer.tsx#L119) an example.

Reply:
Ah, TextTruncated is perfect for this. Thanks.